apps/emacs/emacs.py

```diff
@@ -0,0 +1,328 @@
+import logging
+
+from talon import Context, Module, actions
+
+mod = Module()
+setting_meta = mod.setting(
+    "emacs_meta",
+    type=str,
+    default="esc",
+    desc="""What to use for the meta key in emacs. Defaults to 'esc', since that should work everywhere. Other options are 'alt' and 'cmd'.""",
+)
+
+ctx = Context()
+ctx.matches = r"""
+app: Emacs
+app: emacs
```

Comment:
Would recommend we define an 'emacs' app somewhere and then just use "app: emacs" everywhere rather than repeating this in multiple places. On Mac at least we should consider using a bundle matcher (`app.bundle: org.gnu.Emacs`)

Reply:
done.